### PR TITLE
47659 MSI accessible only via the WAF

### PR DESCRIPTION
**Pull request by tejasi15072**
This PR contains changes related to MSI WAF and webapp firewall rules.

### CODE REVIEW

**Review by mattwin1 — APPROVED** [review]
Looks good to me, and confirmed that access was tested from WAF and otherwise and performed as required. Approved.

File: Deployment/terraform_conditional_run.ps1 [hunk 1]
```diff
@@ -52,9 +52,9 @@ Write-output "Terraform output as json"
 $terraformOutput = terraform output -json | ConvertFrom-Json
 
 write-output "Set JSON output into pipeline variables"
-Write-Host "##vso[task.setvariable variable=MsiWebUrl;isOutput=true]$env:SERVICE_DNS_URL"
+Write-Host "##vso[task.setvariable variable=MsiWafUrl;isOutput=true]$env:SERVICE_DNS_URL"
+Write-Host "##vso[task.setvariable variable=AdminMsiWafUrl;isOutput=true]$env:SERVICE_DNS_URL/RadioNavigationalWarningsAdmin"
```
Comment (barrie-cooper) [inline Deployment/terraform_conditional_run.ps1:56]: Suggest swapping order of "msi" & "admin" as per our discussion earlier for the new URL of the admin site.
```suggestion
Write-Host "##vso[task.setvariable variable=MsiAdminWafUrl;isOutput=true]$env:SERVICE_DNS_URL/RadioNavigationalWarningsAdmin"
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -346,8 +346,8 @@ stages:
         pool: $(WindowsPool)
         displayName: "Dev Functional Automated And Accessibility Tests"
         variables: 
-         url: $[ dependencies.DevDeployTerraform.outputs['DevDeployTerraform.TerraformDeploy.WebsiteURL']  ]
-         rnwAdminUrl: $[ dependencies.DevDeployTerraform.outputs['DevDeployTerraform.TerraformDeploy.WebsiteAdminURL']  ]
+         url: $[ dependencies.DevDeployTerraform.outputs['DevDeployTerraform.TerraformDeploy.MsiWafUrl']  ]
+         rnwAdminUrl: $[ dependencies.DevDeployTerraform.outputs['DevDeployTerraform.TerraformDeploy.AdminMsiWafUrl']  ]
```
Comment (barrie-cooper) [inline azure-pipelines.yml:350]:
```suggestion
         rnwAdminUrl: $[ dependencies.DevDeployTerraform.outputs['DevDeployTerraform.TerraformDeploy.MsiAdminWafUrl']  ]
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -405,8 +405,8 @@ stages:
         pool: $(WindowsPool)
         displayName: "QA Functional Automated And Accessibility Tests"
         variables: 
-         url: $[ dependencies.QADeployTerraform.outputs['QADeployTerraform.TerraformDeploy.WebsiteURL']  ]
-         rnwAdminUrl: $[ dependencies.QADeployTerraform.outputs['QADeployTerraform.TerraformDeploy.WebsiteAdminURL']  ]
+         url: $[ dependencies.QADeployTerraform.outputs['QADeployTerraform.TerraformDeploy.MsiWafUrl']  ]
+         rnwAdminUrl: $[ dependencies.QADeployTerraform.outputs['QADeployTerraform.TerraformDeploy.AdminMsiWafUrl']  ]
```
Comment (barrie-cooper) [inline azure-pipelines.yml:409]:
```suggestion
         rnwAdminUrl: $[ dependencies.QADeployTerraform.outputs['QADeployTerraform.TerraformDeploy.MsiAdminWafUrl']  ]
```

**Review by barrie-cooper — COMMENTED** [review]
Just one suggestion really

**Review by barrie-cooper — APPROVED** [review]
LGTM, thanks @tejasi15072 

Will merge once FTs are working

**Review by usha13529 — APPROVED** [review]
Thanks @tejasi15072, @Shirin14926 & @jigar15426 for changes. Looks good to me.

**Pull request merged**